schema/DoctrineMigrations/Version20250514135139.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;
+
+/**
+* Auto-generated Migration: Please modify to your needs!
+*/
+final class Version20250514135139 extends LoggableMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add errorCount into ';
```

Comment:
This is marked as resolved by the description seems incomplete.
Maybe a missing fix commit?